include/dsn/internal/ports.h

```diff
@@ -123,7 +123,7 @@ namespace dsn
 
 # ifdef _WIN32
 
-// TODO(qinzuoyan) port to windows
+// TODO(qinzuoyan) port to big-endian machine
```

Comment:
the question is: this seems already done for me, i.e., I don't understand why there is a TODO? On the other hand, if this is  a TODO, can you use endian independent functions such as ntoh and hton to fix it?